config/locales/en.yml

```diff
@@ -168,6 +168,8 @@ en:
         label: Finance
       flow:
         label: Flow
+      geography:
+        label: Benefitting recipient geography
```

Comment:
Is this label clear? Has it been tested? I'm not sure I'd know what this meant.

Reply:
I think it is as clear as I could make it right now. It has not been tested no. We are only just getting real users to look at the app this week.

I feel that this says what the value represents right now - early feedback is looking like users do not undersatnd a large amount of the IATI language and terms so that may be a wider issue, would we then change every attribute or change just the content in the user facing part of the app – I think this is one of the many hard question we face.

Reply:
Understood. Happy to leave it for now and see what users think.